qiskit/circuit/library/standard_gates/x.py

```diff
@@ -531,7 +531,7 @@ def inverse(self):
 
 
 class C3XGate(ControlledGate):
-    r"""The 4-qubit controlled X gate.
+    r"""The 3-qubit controlled X gate.
```

Comment:
Good catch, thanks for the fix! But while at it maybe we should make it even clearer. How about
```suggestion
    r"""The X gate controlled on 3 qubits.
```
>